scripts/e2e-test.sh

```diff
@@ -4,8 +4,11 @@ set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
 # new tests should be added before the replica_pod_remove test
-#TESTS="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
-TESTS="install basic_volume_io csi uninstall"
+#DEFAULT_TESTS="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
+DEFAULT_TESTS="install basic_volume_io csi uninstall"
+LONG_TESTS=""
+TESTS=""
```

Comment:
Could you please rename below variables (lowercase) as they're not constants but rather global state and it's confusing to have them together with configuration constants like `DEFAULT_TESTS`. You can also use `readonly VAR=VAL` keyword to enforce. This block of vars (for an outsider) looks confusing.

I would also suggest:
```suggestion
# Global state variables
TESTS=""
```

or something. These glue scripts can get pretty confusing so I would keep them especially readable :-)

Reply:
good point. I will convert all global variables used in this script that are neither exported nor constant to lower-case.

Reply:
Whilst you are at it, probably best to name the variable 'test_list'